### ISSUE TITLE
[Bug]: LC0091 Missing translation for '{0}' in language(s): {1} false positive in report labels

### ISSUE THREAD

**Issue by MODUSCarstenScholling** [ALCops/Analyzers#281]
### Analyzer

LinterCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0091

### Description

LC0091 Missing translation for '{0}' in language(s): {1} false positive in report labels.

<img width="996" height="275" alt="Image" src="https://github.com/user-attachments/assets/5401c5b3-e347-44a6-a4db-8f025d49e4e8" />

### AL code to reproduce

```al
report 50000 "Report Using Labels"
{
    dataset
    {
        dataitem(Loop; Integer)
        {
            DataItemTableView = sorting(Number);
        }

        [...]
    }

    labels
    {
        ImageLbl = 'Image'; // Even after adding a translation, LC0091 is shown
    }
}
```

Original xliff entry
```xml
        <trans-unit id="Report 3254930964 - ReportLabel 2161360639" size-unit="char" translate="yes" xml:space="preserve">
          <source>Image</source>
          <target>Abbildung</target>
          <note from="Developer" annotates="general" priority="2"></note>
          <note from="Xliff Generator" annotates="general" priority="3">Report M365 DX Media/Set - ReportLabel ImageLbl</note>
        </trans-unit>
```

**Comment by Arthurvdv** [comment]
Thanks for reporting this, where with the release of `v0.8.3` of ALCops I think I've resolved this.

Could you verify with this version and let me know if the issue is fixed?

**Comment by MODUSCarstenScholling** [comment]
Yes, sir! 0.8.3 fixes the issue. Thank you Arthur!

**Comment by Arthurvdv** [comment]
Awesome!
